.github/workflows/ci.yml

```diff
@@ -8,8 +8,8 @@ on:
       - master
 
 jobs:
-  test-forge:
-    name: Test Forge
+  test-forge-unit-and-integration:
```

Comment:
hmm, why unit and integration tests are together? couldn't we separate them into two jobs?

Reply:
Units tests are really fast so I don't think we benefit a lot from splitting them